modules/KalturaSupport/components/live/liveCore.js

```diff
@@ -460,6 +460,10 @@
             if ( mw.isIOS8_9() ) {
                 requestObj.rnd = Math.random();
             }
+            		
+            		var oldExpiry = _this.getKalturaClient().baseParam.expiry;
+			_this.getKalturaClient().baseParam.expiry = 5; //Setting is live expiry to 5 seconds
+			
```

Comment:
the cache header is determined  on backend and the browser will cache it for 10 seconds... regardless what you put here